.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,47 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+  pull_request:
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
```

Could we have the same spaces between the sections?

```suggestion
name: golangci-lint

on:
  push:
    tags:
      - v*
    branches:
      - master
      - main
  pull_request:
  
permissions:
  contents: read
  # Optional: allow read access to pull request. Use with `only-new-issues` option.
  # pull-requests: read
  
jobs:
```